api/v1beta1/externaldns_types.go

```diff
@@ -329,7 +325,7 @@ type ExternalDNSBlueCatProviderOptions struct {
 	// }
 	//
 	// See
-	// https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/bluecat.md
+	// https://github.com/kubernetes-sigs/external-dns/blob/226dbb931f7a2019810b3703aec096c4ea4f40ea/docs/tutorials/bluecat.md#using-json-configuration-file
 	// for more information on the necessary configuration values and how to obtain them.
```

Comment:
The linked documentation just seems to say which fields are required and which are not, but not how to obtain them.  I think this change is fine since the goal is just to fix the link, but as I mentioned in my other comment, it would be good in the future to link to product documentation instead.

Reply:
Right, this one was really for consistency (with Azure). Btw [BlueCat is going away](https://github.com/kubernetes-sigs/external-dns/pull/4719) so we don't need to bother about it.